src/main/java/net/dries007/tfc/CommonEventHandler.java

```diff
@@ -203,6 +243,16 @@ public static void onRightClickItem(PlayerInteractEvent.RightClickItem event)
         IPlaceableItem placeable = IPlaceableItem.Impl.getUsable(stack);
         if (placeable != null)
         {
+            if (IPlaceableItem.Impl.getPlaceable(stack) != null)
```

Comment:
This should not be necessary. `onRightClickItem` is fired through a totally separate call path, when you right click without targeting a block (that's within your reach distance). This also (again) ignores the standard placement logic.

Reply:
Unfortunately, it is. I've added it there since both are called at the same time if you are looking at a block (ie: looking at a chest while right clicking with a clay in hand tried to open both GUIs).

Reply:
It's not. There are two seperate call paths - `processRightClickBlock`, and `processRightClick`. One for each event. As in the javadoc:
```
Note that this is NOT fired if the player is targeting a block {@link RightClickBlock} or entity {@link EntityInteract} {@link EntityInteractSpecific}.
```

Reply:
Well, in my testing it said otherwise. I've added that because it happened. Unless something is wrong in my env, that's why I need you to test it to be sure.